libsawtooth/src/journal/block_validator.rs

```diff
@@ -389,7 +389,11 @@ impl BlockValidator {
                 "Unable to receive block validation result: {}",
                 err
             ))),
-        }
+        };
+
+        self.block_scheduler.done(block.block().header_signature());
```

Comment:
The reason for this change should be in the commit message (it is currently in the PR description only).

Reply:
Updated message